src/__test__/home.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../redux/store';
+import Home from '../components/Games/Home';
+
+describe('Test for Missions components', () => {
+  it('Check if the Missions section are available', () => {
+    const website = renderer.create(
+      <Provider store={store}>
+        <Home />
+      </Provider>,
+    );
+    expect(website).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] I think writing just one test is not sufficient to validate the proper running of your application. Please write more tests. You can write unit tests for several components of the app and some pure functions in Redux code 👍🏽 .